test/tests/functional/pbs_reservations.py

```diff
@@ -914,46 +916,60 @@ def test_advance_resv_with_multinode_job_array(self):
         subjid = []
         for i in range(1, 6):
             subjid.append(j.create_subjob_id(jid, i))
-
+        # Wait for reservation to be in running state
```

Comment:
Turn this comment into a self.logger.info().  It is nicer for the test runner to know why they are waiting 30s.  It also serves the same purpose as the comment.